superset/connectors/sqla/views.py

```diff
@@ -269,18 +270,30 @@ class TableModelView(DatasourceModelView, DeleteMixin, YamlExportMixin):  # noqa
         'hive_partitions': _('Hive Partitions')
     }
 
+    def get_table_columns(self,columns):
+        table_columns = []
+        for column in columns:
+            tablecolumn = TableColumn()
```

Comment:
can we rename `tablecolumn` as `tableColumn`?

Reply:
as per superset python files naming  convention ,we can rename it to table_column

Reply:
done